src/components/Dropdown/__stories__/interactions.js

```diff
@@ -0,0 +1,26 @@
+import { expect } from "@storybook/jest";
+import {
+  getByRole,
+  getByText,
+  clickElement,
+  testFunctionWrapper,
+  typeText,
+  getByClassName
+} from "../../../__tests__/interactions-helper";
+
+export const selectAndClearTest = testFunctionWrapper(async canvas => {
+  const dropdownElement = await getByRole(canvas, "textbox");
+  // Open the dropdown
+  await clickElement(dropdownElement);
+  // Filter it
+  await typeText(dropdownElement, "Option");
+  // Select the option
+  const optionToSelect = getByText(canvas, "Option 1");
+  await clickElement(optionToSelect);
+  // click the clear button
+  const clearButton = getByClassName("clear-indicator");
```

Comment:
is there a different locator we can use instead of classname? perhaps role, text or type?